src/cmdc/datasets/uscensus/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP SCHEMA IF EXISTS uscensus CASCADE;
+
+CREATE SCHEMA uscensus;
+
+/* Create and define metadata tables */
+CREATE TABLE uscensus.acs_variables (
+    "id" SERIAL UNIQUE,
+    "year" INTEGER,
+    "product" CHAR(4),
+    "census_id" VARCHAR(20),
+    "label" VARCHAR(350),
+    PRIMARY KEY ("year", "product", "census_id")
+);
+COMMENT ON TABLE uscensus.acs_variables IS E'
+Table Description:
+
+This table contains information on the variables that are listed for one of the ACS products (1 and 5 year estimates):
+
+For more information on these variables, please refer to the US Census page about the [ACS 1 year estimates](https://www.census.gov/data/developers/data-sets/acs-1year.html) and the [ACS 5 year estimates](https://www.census.gov/data/developers/data-sets/acs-5year.html)
+
+Words of caution:
+
+Source: US Census';
+
+COMMENT ON COLUMN uscensus.acs_variables.id IS E'An internal identifier for referencing different variables.';
+COMMENT ON COLUMN uscensus.acs_variables.year IS E'The year this product was collected/published';
+COMMENT ON COLUMN uscensus.acs_variables.product IS E'Which ACS product this variables comes from. Can take the values `acs1` or `acs5`';
+COMMENT ON COLUMN uscensus.acs_variables.census_id IS E'The variable name according to the Census documentation.';
+COMMENT ON COLUMN uscensus.acs_variables.label IS E'A short label that describes what the variable includes.';
+
+
+CREATE TABLE uscensus.bfs_variables (
+    "id" INTEGER,
+    "census_id" VARCHAR(10)
+);
+COMMENT ON TABLE uscensus.bfs_variables IS E'
+Table Description:
+
+This table contains information on the variables that are listed for the weekly Business Formation Statistics product
+
+The Business Formation Statistics (BFS) are an experimental data product of the U.S. Census Bureau developed in research collaboration with economists affiliated with Board of Governors of the Federal Reserve System, Federal Reserve Bank of Atlanta, University of Maryland, and University of Notre Dame. The BFS provide timely and high frequency information on new business applications and formations in the United States. More information on Census Bureau Experimental Data products can be found [here](https://www.census.gov/data/experimental-data-products.html)
+
+Words of caution: This data product is experimental! The US Census has back-filled it to 2006 but there is no guarantee that they continue to publish it.
+
+Source: US Census';
+
+
+/* Create and define data tables */
+CREATE TABLE uscensus.acs_data (
+    "id" INTEGER REFERENCES uscensus.acs_variables(id),
+    "fips" INTEGER,
```

Comment:
FIPS should reference our data.us_counties table, with caveat that we might change and combine data.us_states and data.us_counties into a single data.us_fips table. The point is this needs to have a foreign key so we can link it up with all the other county level variables

Reply:
`fips` can be:

* `0` for entire nation
* 2 digit fips for state
* 5 digit fips for counties
* 11 digit fips for tracts

I think we eventually need the `us_fips` table to include all of these geographies.

For now, I guess we can just choose not to collect data on tracts, but I think it's worth having the `us_fips` table cover all 4 geographies at some point.

Reply:
I'll have it reference `data.us_fips` for now but, in the relatively near future, we should create a `us_fips` table that has at least counties and states.